src/components/OrganizationSaas/Dashboard/Create.tsx

```diff
@@ -1,24 +1,167 @@
+              justifyContent='center'
+              gridTemplateColumns='repeat(5, 1fr)'
+              gridRowGap='50px'
+              borderRadius='xl'
+              filter='grayscale(100%)'
+              py='10px'
+              pr='30px'
+              mb={{ base: '50px', lg: 0 }}
+            >
+              <Flex alignItems='center' ml='30px'>
+                <Image src={barca} h={'55px'} />
+              </Flex>
+              <Flex alignItems='center' justifyContent='center'>
+                <Image src={omnium} h={'60px'} />
+              </Flex>
+              <Flex alignItems='center' justifyContent='center'>
+                <Image src={berga} h={'50px'} />
+              </Flex>
+              <Flex alignItems='center' justifyContent='center'>
+                <Image src={bisbal} h={'45px'} />
+              </Flex>
+              <Flex alignItems='center'>
+                <Image src={coec} h={'40px'} />
+              </Flex>
+              <Flex alignItems='center' ml='30px'>
+                <Image src={erc} h={'35px'} />
+              </Flex>
+              <Flex alignItems='center' pl='10px' justifyContent='center'>
+                <Image src={bellpuig} h={'50px'} />
+              </Flex>
+              <Flex alignItems='center' justifyContent='center'>
+                <Image src={ticanoia} h={'30px'} />
+              </Flex>
+              <Flex alignItems='center' justifyContent='center'>
+                <Image src={decidim} h={'40px'} />
+              </Flex>
+              <Flex alignItems='center'>
+                <Image src={bloock} h={'30px'} />
+              </Flex>
+            </Grid>
+          </Box>
+          <Flex flexDirection='column' alignItems={{ base: 'center', lg: 'start' }} mb='100px' w='full'>
+            <Text fontSize={{ base: '28px', md: '48px', lg: '32px', xl: '48px' }} color='white'>
+              The global voting platform
+            </Text>
+            <Text fontSize='19px' color='white' textAlign={{ base: 'center', lg: 'start' }}>
+              Cut cost, Save Time: Secure, Private, and GDPR Compliant Voting
+            </Text>
+          </Flex>
+        </Flex>
+        <Flex mt='auto' zIndex='3' flexDirection='column' alignItems='center' justifyContent='center'>
+          <List display='flex'>
+            <ListItem
+              me={{
+                base: '20px',
+                md: '44px',
+              }}
+            >
+              <Link fontWeight='500' href='mailto:hello@simmmple.com'>
+                Support
+              </Link>
+            </ListItem>
+            <ListItem
+              me={{
+                base: '20px',
+                md: '44px',
+              }}
+            >
+              <Link fontWeight='500' href='https://simmmple.com/terms-of-service'>
+                Terms of Use
+              </Link>
+            </ListItem>
+            <ListItem>
+              <Link fontWeight='500' href='https://www.blog.simmmple.com/'>
```

Comment:
Mailto and this two urls should point to the correct place better

Reply:
Also, if it's an external link, always use `isExternal`.